instances/k8sworker/scripts/setup.template.sh

```diff
@@ -83,6 +83,25 @@ if [[ ! -z "$${NVMEDEVS}" ]]; then
     fi
 fi
 
+## Login iSCSI volume mount and create filesystem
+######################################
+iqn=$(iscsiadm --mode discoverydb --type sendtargets --portal 169.254.2.2:3260 --discover| cut -f2 -d" ")
+
+if [ -n "$${iqn}" ]; then
+    echo "iSCSI Login $${iqn}"
+    iscsiadm -m node -o new -T $${iqn} -p 169.254.2.2:3260
+    iscsiadm -m node -o update -T $${iqn} -n node.startup -v automatic
+    iscsiadm -m node -T $${iqn} -p 169.254.2.2:3260 -l
+    # Wait for device to apear...
+    until ls "/dev/disk/by-path/ip-169.254.2.2:3260-iscsi-$${iqn}-lun-1"; do sleep 1 && echo -n "."; done
```

Comment:
`until [[ -e "/dev/disk/by-path/ip-169.254.2.2:3260-iscsi-$${iqn}-lun-1" ]]; do` if you care about the output..?

Reply:
yeah I had "file -E" test but '-E' is not on OEL and wasn't really thinking. Much better.